src/browser/sidebar/tab.js

```diff
@@ -98,7 +98,9 @@ const styleSheet = StyleSheet.create({
     position: 'absolute',
     paddingLeft: '10px',
     right: 0,
-    top: 0
+    top: 0,
+    transition: `transform 400ms cubic-bezier(0.215, 0.610, 0.355, 1.000),
+        opacity 300ms ease-out`
```

Comment:
nit: the spacing should line up with `transform`, or else maybe be on the same line.